### PR TITLE
Change some texts on the homepage

### PR DESCRIPTION
**Pull request by ann0see**
Jamulus is currently bad at SEO. I think we could improve the wording on the homepage

### CODE REVIEW

File: 1-index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Jamulus ‒ Play music online. With friends. For free."
+title: "Jamulus ‒ Jam online. With friends. For free."
```
Comment (gilgongo) [inline 1-index.html:2]: From an SEO point of view, I agree that "jam" is probably going to be a more realistic term for us to target than "play". It's just a pity that perhaps only a small subset of people actually use Jamulus to jam with.

**Review by gilgongo — COMMENTED** [review]
Looks good. As I've pointed out before, tinkering with the home page content won't make any difference to our ranking at this point though, but there's no harm in it :-)

**Comment by ann0see** [comment]
> As I've pointed out before, tinkering with the home page content won't make any difference to our ranking at this point though

Not 100% sure about that, but we can try.

**Comment by gilgongo** [comment]
> Not 100% sure about that, but we can try.

Hehe. Well, you can't be 100% sure about anything in SEO :-)

**Pull request merged**